packages/react-server/core/util/PageUtil.js

```diff
@@ -183,11 +183,11 @@ function standardizeDebugComments(debugComments) {
 }
 
 function standardizeMetaTags(metaTags) {
-	return PageUtil.makeArray(metaTags).map(metaTag => Q(metaTag));
+	return PageUtil.makeArray(metaTags).map(metaTag => Promise.resolve(metaTag));
```

Comment:
Come to think of it I'm already using `Promise.resolve` unbound all over the place.

So I guess it works. :D